compliance_checker/suite.py

```diff
@@ -42,6 +42,38 @@ class CheckSuite(object):
     def __init__(self):
         self.col_width = 80
 
+    # @classmethod
+    def priorityheader(self, check):
+        """
+        Method to determine which scoring terminology to use for the headers.
+        @param check  name of the check being run
+
+        @return headerout  dict of header priorities
+        """
+        headermap = {1:  {3: 'Highly Recommended',
+                          2: 'Recommended',
+                          1: 'Suggested'},
+                     2:  {3: 'Errors',
+                          2: 'Warnings',
+                          1: 'Info'},
+                     3:  {3: 'Required',
+                          2: 'Recommended',
+                          1: 'Suggested'},
+                     4:  {3: 'High Priority',
+                          2: 'Medium Priority',
+                          1: 'Low Priority'}}
+        if check.startswith('acdd') or check.startswith('ioos'):
```

Comment:
@Bobfrat @benjwadams I looked at the Travis CI failed builds and this seems to be the error. I'll make adjustments, but I'll wait for your comments to push another commit

Reply:
I think you still want to check for None here. I'll merge and make the small fix in a subsequent PR to the refactor-scoring branch